parsec/backend/organization.py

```diff
@@ -79,7 +79,9 @@ class OrganizationStats:
     data_size: int
     metadata_size: int
     users: int
+    active_users: int
     workspaces: int
+    users_per_profile_detail: Dict[str, Counter]
```

Comment:
key should be of type `UserProfile`

Reply:
See my comment about `organization_stats` command schema: we'd better use more explicit types here

```python
@attr.s(slots=True, frozen=True, auto_attribs=True)
class UsersPerProfileDetailItem:
    active: int
    revoked: int

@attr.s(slots=True, frozen=True, auto_attribs=True)
class OrganizationStats:
    [...]
    users_per_profile_detail: List[UsersPerProfileDetailItem]

```